lib/core-data/get-variations.js

```diff
@@ -0,0 +1,94 @@
+import _ from 'lodash';
+import logger from '../utils/log';
+
+const log = logger(__filename);
+
+/**
+ * adds a 'componentVariation' field to the components schema
+ * @param {object} schema
+ * @param {array} componentVariations
+ * @returns {object}
+ **/
+function addField(schema, componentVariations) {
+  const variations = _.map(componentVariations, function (variant) {
+      return {
+        name: _.capitalize(variant.split('_')[1]),
```

Comment:
do we care to replace dash-separated names with space-separated?

Reply:
The underscore separates the variation from the component name. They're pulled originally from the css filenames.